engine/image-loader/packages.lisp

```diff
@@ -2,10 +2,14 @@
 
 (defpackage :lbge.image-loader
   (:use :cl)
-  (:export :load-image :make-image)
+  (:export :load-image)
   (:import-from :alexandria
                 :switch))
 
+(defpackage :lbge.image-loader.image
+  (:use :cl)
+  (:export :make-image :width :height :channels :data))
```

Comment:
I think it's not very good to export such generic names as `width`, `height` or `data`.
More appropriate names are `image-width`, etc.